README.md

```diff
@@ -127,6 +127,16 @@ MultiplePermissionsListener dialogMultiplePermissionsListener = /*...*/;
 Dexter.checkPermissions(new CompositePermissionListener(snackbarMultiplePermissionsListener, dialogMultiplePermissionsListener, /*...*/), Manifest.permission.CAMERA, Manifest.permission.RECORD_AUDIO);
 ```
 
+** If your application has to support configuration changes based on screen rotation remember to add a call to ``Dexter`` in your Activity ``onCreate`` method as follows:**
```

Comment:
Add a header at the same level as Single permission/Multiple permissions to explain this as it's something that will be useful for both. Something like **Handling Activity rotation**.